gammapy/estimators/tests/test_flux_point_estimator.py

```diff
@@ -232,6 +232,8 @@ def test_run_pwl(fpe_pwl):
     actual = table["npred_null"].data
     assert_allclose(actual, [[942.], [398.], [14.]])
 
+    actual = table.meta["UL_CONF"]
+    assert_allclose(actual, 0.9544997)
```

Comment:
+1 To keep a higher precision here, GADF does not specify anything...(https://gamma-astro-data-formats.readthedocs.io/en/latest/spectra/flux_points/index.html#header-keywords)